receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -158,6 +167,13 @@ metrics:
     unit: "{transactions}/s"
     gauge:
       value_type: double
+  sqlserver.database_io.read_latency:
+    enabled: false
+    description: Total time, in milliseconds, that the users waited for reads.
+    unit: "ms"
```

Comment:
Should we emit seconds in float instead? The basic units are recommended in the OTel spec.

Reply:
Good point, I missed that. [Relevant section of the spec](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units) 

I've updated.